packages/block-library/src/columns/edit.js

```diff
@@ -60,13 +60,34 @@ function ColumnsEditContainer( {
 } ) {
 	const { isStackedOnMobile, verticalAlignment, templateLock } = attributes;
 
-	const { count, canInsertColumnBlock } = useSelect(
+	const { count, canInsertColumnBlock, minCount } = useSelect(
 		( select ) => {
+			const {
+				canInsertBlockType,
+				canRemoveBlock,
+				getBlocks,
+				getBlockCount,
+			} = select( blockEditorStore );
+			const innerBlocks = getBlocks( clientId );
+
+			// Get the indexes of columns for which removal is prevented.
```

Comment:
Tiny nit: when reading this, at first I thought there might be a reason we needed to get all the indexes, rather than just the highest index. Would it be worth clarifying that in this comment?
```suggestion
			// Get the indexes of columns for which removal is prevented.
			// The highest index will be used to determine the minimum column count.			
```

Reply:
I think my suggestion comment here included some whitespace that tripped up Prettier in the Github Action. Sorry about that!